cinema-backend/src/main/java/pl/edu/pw/ee/cinemabackend/config/jwt/JwtServiceImpl.java

```diff
@@ -52,7 +50,14 @@ public final String generateRefreshToken(UserDetails userDetails) {
 
     @Override
     public final String generateToken(UserDetails userDetails) {
-        return generateToken(Collections.emptyMap(), userDetails, expirationTime);
+        Map<String, Object> claims = Collections.emptyMap();
+        if(userDetails instanceof User user) {
```

Comment:
user implementuje userDetails lepiej to po prostu przekazać jako authorities user'a